thunder/torch/__init__.py

```diff
@@ -5620,6 +5621,13 @@ def linear(a: TensorLike, w: TensorLike, /, bias: None | TensorLike = None) -> T
     return prims.linear(a, w, bias)
 
 
+if LooseVersion(torch.__version__) >= "2.8":
+
+    @torchsymbol(torch._grouped_mm)
+    def _grouped_mm(a: TensorProxy, b: TensorProxy, offs: TensorProxy) -> TensorProxy:
```

Comment:
It seems that `torch._grouped_mm` can take bias and out_dtype as per https://github.com/pytorch/pytorch/blob/c6b479bc09090336dcd1c97f5e368e76490065ea/tools/autograd/derivatives.yaml#L1209. Could you match the signature?

Reply:
Thanks for the pointer! 

> Could you match the signature?

Yes. However, in this PR, I don't plan to support bias or out_dtype so `utils.check`. This is for YAGNI. Sounds good to you?

Reply:
I think my ask is to update torchex to have `grouped_mm` and move the check of `bias` and `dtype` to `_grouped_mm_check_` of nvfuserex_impl so that the coverage would be a bit clearer.
Would it sound reasonable?

Reply:
Following the YAGNI principle, I recommend we do that in the future. It's unclear to me what the semantics are when offsets=None and/or bias!=None. So it's unclear to me how to write [_grouped_mm_meta](https://github.com/Lightning-AI/lightning-thunder/pull/2326/files#diff-16f33141471a03112ba59caeb9fc36095d5df9b1564e79994e63d8d4ac175606R3767) for those cases. I'd rather we keep the Thunder prims simple for the cases we need. Also note that `torch._grouped_mm` is an experimental API so it's more likely to change than other APIs.

Reply:
okay, understood.